internal/pkg/service/buffer/sink/tablesink/storage/repository/file_repo.go

```diff
@@ -352,6 +352,7 @@ func (r *FileRepository) rotateAllIn(rb rollback.Builder, now time.Time, parentK
 					txn.Merge(r.updateTxn(oldFile, modified))
 				} else {
 					errs.Append(err)
+					continue
```

Comment:
There is a for loop to rotate files/slice in each sink.

There were missing `continue` calls:
- Now it doesn't causes problems, most of the possible errors are tested.
- But it is not OK to continue in code on error.
- There may be a combination of errors resulting to a nil pointer panic, for example.